lms/djangoapps/course_api/blocks/views.py

```diff
@@ -183,7 +183,7 @@ class BlocksView(DeveloperErrorViewMixin, ListAPIView):
             Returned only if "show_correctness" is included in the "requested_fields" parameter.
     """
 
-    def list(self, request, usage_key_string):  # pylint: disable=arguments-differ
+    def list(self, request, usage_key_string, hide_access_denials=False):  # pylint: disable=arguments-differ
```

Comment:
@cpennington should we be able to pass hide_access_denials=true for api v2?  
if so, noting that still need to fix probably by modifying the course blocks form, as that is not currently working

Reply:
No, I don't think we need to leave that as a usable parameter on a call to api v2 (which I think is the behavior that you'll get as is).